### PR TITLE
Market app

### PR DESCRIPTION
**Pull request by fleafclover**
I changed the paths of the api s in app.py and paths in html files. Now it is ready to be merged to master branch. Please review it again.

### CODE REVIEW

**Review by gulsahc — COMMENTED** [review]
I think variables 'client_id' and 'secret' are not necessary. Other than that your code seems fine. Do not forget to include your unit tests.

**Review by OnurSefa — APPROVED** [review]
I see that my suggestions related to this branch have been accomplished. The suggestion can be tracked from #84 pull request. Thanks for your great work.

File: app.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import pymongo
+
+
+
+app = Flask(__name__)
+
+
+client = pymongo.MongoClient("mongodb+srv://base_user:base_user_password@cluster0.dbcb9.mongodb.net/first")
+db = client.first
+collection = db.berkay
```
Comment (safaaltunel) [inline app.py:18]: you can change this variable name because it may cause a conflict. recommended name is something like market_collection

Comment (fleafclover) [reply app.py:18]: I think my file organization can be problematic as well. I will change it and create a new pull request.

**Pull request closed without merging**